### PR TITLE
Get ntp config

### PR DESCRIPTION
**Pull request by cheburakshu**
ref: #vrae-3229

### CODE REVIEW

File: src/saltext/vmware/modules/esxi.py [hunk 1]
```diff
@@ -704,6 +704,70 @@ def get_dns_config(
...
+        datacenter_name=datacenter_name,
+        get_all_hosts=host_name is None,
+    )
+
+    try:
+        for h in hosts:
+            ntp_config = h.configManager.dateTimeSystem
+            if not ntp_config:
+                continue
+            ret[h.name] = {
```
Comment (waynew) [inline src/saltext/vmware/modules/esxi.py:749]: **suggestion** could flip this:

```
if ntp_config:
    ret[h.name] = {...}
```

File: src/saltext/vmware/modules/esxi.py [hunk 2]
```diff
@@ -704,6 +704,70 @@ def get_dns_config(
...
+                "time_zone": ntp_config.dateTimeInfo.timeZone.key,
+                "time_zone_name": ntp_config.dateTimeInfo.timeZone.name,
+                "time_zone_description": ntp_config.dateTimeInfo.timeZone.description,
+                "time_zone_gmt_offset": ntp_config.dateTimeInfo.timeZone.gmtOffset,
+                "ntp_servers": list(ntp_config.dateTimeInfo.ntpConfig.server),
+                "ntp_config_file": list(ntp_config.dateTimeInfo.ntpConfig.configFile)
+                if ntp_config.dateTimeInfo.ntpConfig.configFile
+                else None,
+            }
+        return ret
```
Comment (waynew) [inline src/saltext/vmware/modules/esxi.py:759]: **suggestion** this early return is unnecessary either with or without changing the previous block.

File: tests/integration/modules/test_esxi.py [hunk 3]
```diff
@@ -457,3 +457,34 @@ def test_esxi_get(service_instance):
...
+        assert sorted(list(ret[host])) == sorted(
+            [
+                "ntp_config_file",
+                "time_zone",
+                "time_zone_description",
+                "time_zone_name",
+                "ntp_servers",
+                "time_zone_gmt_offset",
+            ]
+        )
```
Comment (waynew) [inline tests/integration/modules/test_esxi.py:482]: **question** can you use the test_value_scraper to grab the expected values, and then assert that the expected == actual? When we shift to actually configuring the ntp values on the hosts for the tests, this test wouldn't really have to be refactored at that point.

**Review by waynew — CHANGES_REQUESTED** [review]
Looks good - just some minor changes here

File: tests/integration/modules/test_esxi.py (hunk 3 above)
Comment (cheburakshu) [reply tests/integration/modules/test_esxi.py:482]: ok

File: src/saltext/vmware/modules/esxi.py (hunk 2 above)
Comment (cheburakshu) [reply src/saltext/vmware/modules/esxi.py:759]: ok

File: src/saltext/vmware/modules/esxi.py (hunk 1 above)
Comment (cheburakshu) [reply src/saltext/vmware/modules/esxi.py:749]: ok

**Review by waynew — APPROVED** [review]
:+1:

**Pull request merged**